Sources/LiveViewNative/Property Wrappers/FormState.swift

```diff
@@ -117,7 +117,7 @@ public struct FormState<Value: FormValue> {
                 return value
             case .form(let formModel):
                 guard let elementName = element.attributeValue(for: "name") else {
-                    fatalError("Expected @FormState in form mode to have element with name")
+                    return initialValue
```

Comment:
Log a warning in this case here and in the setter. This shouldn't be reachable unless the `name` attribute on an element is dynamic (since `resolveMode` validates the presence of the attribute), but in case it happens, it shouldn't fail silently